README.md

```diff
@@ -1,73 +1,86 @@
-pmdk: Persistent Memory Development Kit
-=======================================
+# **PMDK: Persistent Memory Development Kit**
 
 [![Build Status](https://travis-ci.org/pmem/pmdk.svg?branch=master)](https://travis-ci.org/pmem/pmdk)
 [![Build status](https://ci.appveyor.com/api/projects/status/u2l1db7ucl5ktq10/branch/master?svg=true&pr=false)](https://ci.appveyor.com/project/pmem/pmdk/branch/master)
 [![Coverity Scan Build Status](https://img.shields.io/coverity/scan/3015.svg)](https://scan.coverity.com/projects/pmem-pmdk)
 [![PMDK release version](https://img.shields.io/github/release/pmem/pmdk.svg)](https://github.com/pmem/pmdk/releases/latest)
 [![Coverage Status](https://codecov.io/github/pmem/pmdk/coverage.svg?branch=master)](https://codecov.io/gh/pmem/pmdk/branch/master)
 
-This is the top-level README.md of the Persistent Memory Development Kit.
-For more information, see http://pmem.io.
 
-Bugs and feature requests are tracked in our [GitHub Issues Database](https://github.com/pmem/issues/issues).
+The **Persistent Memory Development Kit (PMDK)** is a collection of libraries and tools for System Administrators and Application Developers to simplify managing and accessing persistent memory devices. For more information, see http://pmem.io.
 
-### The Libraries ###
+To install PMDK libraries, either install pre-built packages, which we build for every stable release, or clone the tree and build it yourself. **Pre-built** packages can be found in popular Linux distrubution package repositories, or you can check out our recent stable releases on our [github release page](https://github.com/pmem/pmdk/releases). Specific installation instructions are outlined below.
 
-Please see the file [LICENSE](https://github.com/pmem/pmdk/blob/master/LICENSE)
-for information on how this library is licensed.
+Bugs and feature requests for this repo are tracked in our [GitHub Issues Database](https://github.com/pmem/issues/issues).
 
-This tree contains a collection of libraries for using Non-Volatile Memory
-(NVM).  There are currently nine libraries:
+## Contents
+1. [Libraries and Utilities](#libraries-and-utilities)
+3. [Version Conventions](#version-conventions)
+4. [Pre-Built Packages for Windows](#pre-built-packages-for-windows)
+5. [Dependencies](#dependencies)
+	* [Linux](#linux)
+	* [Windows](#windows)
+	* [FreeBSD](#freebsd)
+6. [Building PMDK on Linux or FreeBSD](#building-pmdk-on-linux-or-freebsd)
+	* [Make Options](#make-options)
+	* [Testing Libraries](#testing-libraries-on-linux-and-freebsd)
+	* [Memory Management Tools](#memory-management-tools)
+6. [Building PMDK on Windows](#building-pmdk-on-windows)
+	* [Testing Libraries](#testing-libraries-on-windows)
+4. [Getting Started](#getting-started)
```

Comment:
Hmmm... should we maybe move the getting started part to the beginning of the list? I'm not sure.

Reply:
Good point, I will move it.